spec/function.dd

```diff
@@ -1248,75 +1248,86 @@ $(H2 $(LNAME2 parameters, Function Parameters))
-    ))
-    $(TROW $(D return), $(ARGS Parameter may be returned or copied to the first parameter,
-    but otherwise does not escape from the function.
-    Such copies are required not to outlive the argument(s) they were derived from.
-    Ignored for parameters with no references.
-    See $(DDSUBLINK spec/memory-safe-d, scope-return-params, Scope Parameters).))
-    $(TROW $(D lazy), argument is evaluated by the called function and not by the caller)
-    $(TROW $(D const), argument is implicitly converted to a const type)
-    $(TROW $(D immutable), argument is implicitly converted to an immutable type)
-    $(TROW $(D shared), argument is implicitly converted to a shared type)
-    $(TROW $(D inout), argument is implicitly converted to an inout type)
-    )
+
+        $(TABLE_2COLS Parameter Storage Class and Type Constructor Overview,
+        $(THEAD Storage Class, Description)
+
+        $(TROW $(I none), The parameter will be a mutable copy of its argument.)
+
+        $(TROW $(D in), The parameter is an input to the function.)
+
+        $(TROW $(D out), The argument must be an lvalue$(COMMA) which will be passed by reference and initialized
+        upon function entry with the default value (`T.init`) of its type.
+        )
+
+        $(TROW $(D ref), The parameter is an $(I input/output) parameter$(COMMA) passed by reference.
+        )
+
+        $(TROW $(D scope), $(ARGS
+        The parameter must not escape the function call
+        (e.g. by being assigned to a global variable).
+        Ignored for any parameter that is not a reference type.
+        ))
+
+        $(TROW $(D return), $(ARGS Parameter may be returned or copied to the first parameter,
+        but otherwise does not escape from the function.
+        Such copies are required not to outlive the argument(s) they were derived from.
+        Ignored for parameters with no references.
+        See $(DDSUBLINK spec/memory-safe-d, scope-return-params, Scope Parameters).))
+
+        $(TROW $(D lazy), argument is evaluated by the called function and not by the caller)
+
+        $(THEAD Type Constructor, Description)
+
+        $(TROW $(D const), argument is implicitly converted to a const type)
+        $(TROW $(D immutable), argument is implicitly converted to an immutable type)
+        $(TROW $(D shared), argument is implicitly converted to a shared type)
+        $(TROW $(D inout), argument is implicitly converted to an inout type)
+        )
+
+$(H3 $(LNAME2 in-params, In Parameters))
+
+        $(P The parameter is an input to the function. Input parameters behaves as if they have
+        the $(D const scope) storage classes. Input parameters may be passed by reference by the compiler.
+        Unlike $(D ref) parameters$(COMMA) $(D in) parameters can bind to both lvalues and rvalues (such as literals).
+        Types that would trigger a side effect if passed by value (such as types with postblit$(COMMA)
```

Comment:
suggested change: "... such as types with copy constructors, postblit ...". Put the copy constructor first so that people see it as a better alternative than postblit.

Reply:
done